libsrc/runtime/div.s

```diff
@@ -1,37 +1,29 @@
 ;
-; Ullrich von Bassewitz, 07.08.1998
+; Christian Krueger, 24-May-2018
 ;
 ; CC65 runtime: division for signed ints
 ;
 
 ; When negating values, we will ignore the possibility here, that one of the
-; values if $8000, in which case the negate will fail.
+; values is $8000, in which case the negate will fail.
 
         .export         tosdiva0, tosdivax
-        .import         popsargs, udiv16, negax
-        .importzp       sreg, tmp1, tmp2
+        .import         absvaludiv16, negax
+        .importzp       sp, ptr1, tmp1
 
 tosdiva0:
         ldx     #0
 tosdivax:
-        jsr     popsargs        ; Get arguments from stack, adjust sign
-        jsr     udiv16          ; Do the division
-        ldx     sreg+1          ; Load high byte of result
-
-; Adjust the sign of the result. tmp1 contains the high byte of the left
-; operand, tmp2 contains the high byte of the right operand.
-
-        lda     tmp1
-        eor     tmp2
-        bpl     Pos             ; Jump if sign of result positive
-
-; Result is negative
-
-        lda     sreg            ; Load low byte of result
-        jmp     negax           ; Adjust the sign
-
-; Result is positive
-
-Pos:    lda     sreg
+        pha                         ; check if high-bytes indicate
```

Comment:
The /libsrc/runtime pretty consistently uses Uppercase for the first letter of comments. Please keep to that style for the sake of consistency.